src/hotspot/share/runtime/objectMonitor.hpp

```diff
@@ -179,7 +171,7 @@ class ObjectMonitor {
   jint  _contentions;               // Number of active contentions in enter(). It is used by is_busy()
```

Comment:
Future RFE - can we replace jint with int32_t or even int or some C++ types.  We're trying not to have Java types leak into runtime code since this doesn't directly interface with Java.

Reply:
It can be a future RFE, but it won't be at the top of my list of
things to do. There may already be an RFE for that.

Reply:
No, I assume it's not high priority.  I'll file an RFE because someday I want these to be cleaned up as a personal nit.

Reply:
Filed the following new RFE:
    JDK-8256306 ObjectMonitor::_contentions field should not be 'jint'
    https://bugs.openjdk.java.net/browse/JDK-8256306